docs/docusaurus/docs/cloud/set_up_gx_cloud.md

```diff
@@ -82,7 +92,84 @@ Currently, the GX Cloud user interface is configured for Snowflake and this proc
 
     If you stop the GX Agent, close the terminal, and open a new terminal you'll need to set the environment variables again.
 
-    To edit an environment variable, stop the GX Agent, edit the environment variable, save the change, and then restart the GX Agent.
+    To edit an environment variable, stop the GX Cloud agent, edit the environment variable, save the change, and then restart the GX Cloud agent.
+
+## Invite a user
+
+1. In GX Cloud, click **Settings** > **Users**.
+
+2. Click **Invite User** and complete the following fields:
+
+    - **Email** - Enter the user's email address.
+
+    - **Organization Role** - Select **Viewer**, **Editor**, or **Admin**. Viewers can view Checkpoints and Validation Results, Editors can create and edit Expectations and can create access tokens, and Admins can perform all GX Cloud administrative functions.
+
+3. Click **Invite**.
+
+    An email invitation is sent to the user.
+
+</TabItem>
+<TabItem value="Viewer/Editor">
+
+When your Admin invites you to an org, you'll need to generate and record your access tokens, set the environment variables to access Data Assets on your Data Source, and then sign in to GX Cloud.
```

Comment:
@Rachel-Reverie, this topic no longer appears in the TOC. I am keeping it temporarily in case it is required when additional Data Sources are supported.